TabloidMVC/Views/Shared/_Layout.cshtml

```diff
@@ -42,6 +42,9 @@
                             <li class="nav-item mx-0 mx-lg-1">
                                 <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" asp-area="" asp-controller="Account" asp-action="Logout">LOGOUT</a>
                             </li>
+                            <li class="nav-item mx-0 mx-lg-1">
+                                <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" asp-area="" asp-controller="Tag" asp-action="Index">TAG Management</a>
+                            </li>
```

Comment:
Please move list item for "TAG Management" to the left of "LOGOUT" on the navbar so that logout is the furthest right link on the navbar